browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -373,6 +379,8 @@ void BraveNewTabMessageHandler::HandleSaveNewTabPagePref(
     settingsKey = kNewTabPageShowAddCard;
   } else if (settingsKeyInput == "showGemini") {
     settingsKey = kNewTabPageShowGemini;
+  } else if (settingsKeyInput == "showBitcoinDotCom") {
```

Comment:
Should be within `#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)`

Reply:
fixed